src/elexmodel/handlers/data/CombinedData.py

```diff
@@ -163,24 +196,77 @@ def _get_unexpected_units(self, aggregates):
 
         return unexpected_units
 
-    def _get_non_modeled_units(self, percent_reporting_threshold, turnout_factor_lower, turnout_factor_upper):
-        expected_geographic_units = self._get_expected_geographic_unit_fips().tolist()
+    def _fit_outlier_detection_model(self, reporting_units, response_variable, outlier_z_threshold):
```

Comment:
Do you have a sense of:

1. how much better this works compared to say, mean ± (`z` * standard deviation) or the IQR method?
2. how much additional runtime this adds on to the model?

🤔

Reply:
1. I never saw how well the mean method works on actual outliers. but this does a very good job flagging outliers (if you run it against MD you can see)
2. constant runtime, cuz we only do it once. but either way. negligible because no bootstrap or cv involved.